CMakeLists.txt

```diff
@@ -64,21 +67,16 @@ endforeach( OUTPUTCONFIG CMAKE_CONFIGURATION_TYPES )
 set(SC_MEMORY_SRC "${SC_MACHINE_ROOT}/sc-memory")
 
 if (${SC_USE_SANITIZER} STREQUAL "address")
-    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address -fno-omit-frame-pointer")
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fno-omit-frame-pointer")
-
-    if(NOT SC_COVERAGE)
-        set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O1")
-        set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O1")
-    endif()
+    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=address -O1 -fno-omit-frame-pointer")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -O1 -fno-omit-frame-pointer")
 elseif (${SC_USE_SANITIZER} STREQUAL "memory")
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=memory -fPIE -pie -fno-omit-frame-pointer")
     set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=memory -fPIE -pie -fno-omit-frame-pointer")
 endif()
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcolor-diagnostics -std=c++17")
-    if (${SC_USE_SANITIZER} STREQUAL "none" AND NOT SC_COVERAGE)
+    if (SC_OPTIMIZATION)
```

Comment:
remove

Reply:
Should I fixup the Review fixes commit to remove it?